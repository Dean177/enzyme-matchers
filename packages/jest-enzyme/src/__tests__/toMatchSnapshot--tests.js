const React = require('react');

describe('toMatchSnapshot', () => {
  it('works with its jest native approach', () => {
    expect(<div />).toMatchSnapshotz();
  });
});
