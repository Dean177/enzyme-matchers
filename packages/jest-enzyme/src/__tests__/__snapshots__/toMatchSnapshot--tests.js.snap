exports[`toMatchSnapshot works with its jest native approach 1`] = `<div />`;
